resources/pdns_recursor_install_rhel.rb

```diff
@@ -66,3 +69,9 @@
     action :remove
   end
 end
+
+action_class.class_eval do
+  def whyrun_supported?
```

Comment:
Fair point. I had assumed this bug was already fixed. You shouldn't have to go all the way to a class eval though.